doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1325,6 +1325,41 @@ Recalculate the UNF value of a dataset version, if it's missing, by supplying th
 
   POST http://$SERVER/api/admin/datasets/integrity/{datasetVersionId}/fixmissingunf
 
+Dataset Validation
```

Comment:
This writeup is great but I would suggest linking it from a future version of http://guides.dataverse.org/en/4.13/admin/dataverses-datasets.html as well. Otherwise, it's a little buried in the very loooooooooooong Native API page. On a related note @dlmurphy recently showed me a trick for linking to an arbitrary location of a page in Sphinx. See 3a8247c for an example, the `:ref:` tag. I'm happy to push a commit to this branch to add this if it's desired.

Reply:
Makes sense. I'll also link to it from the "troubleshooting" API. It was a troubleshooting operation for us; we had these validation errors throughout the app, and we used this new API to find what it was. Maybe it'll help other installations in the future as well.